src/components/VNotificationBanner.vue

```diff
@@ -29,9 +29,10 @@
 <script lang="ts">
 import { defineComponent, PropType } from '@nuxtjs/composition-api'
 
+import { useStorage } from '@vueuse/core'
```

Comment:
I've started regretting my suggestion of using VueUse directly. Now, I get an IDE error saying 'Cannot find module '@vueuse/core' or its corresponding type declarations.'. If I add a Nuxt build module as is suggested in the VueUse docs (`@vueuse/nuxt`, https://vueuse.org/guide/index.html#nuxt), I get an 'Unmet peer dependency Nuxt 3' error. The related issue in the repository is marked as fixed (https://github.com/vueuse/vueuse/issues/2176), but it seems to be fixed for Nuxt Bridge, not Nuxt 2.

Maybe we should wait with VueUse until we have Nuxt 3? 😢 Or should we tolerate the IDE warnings?